src/glibc.zig

```diff
@@ -1119,3 +1138,49 @@ pub fn needsCrtiCrtn(target: std.Target) bool {
         else => true,
     };
 }
+
+// Glibc v2.17 is the oldest supported linking target due to dependencies
+// from the Zig standard library.
+pub fn isSupportedGlibcVersion(ver: Version) bool {
+    return (ver.major > 2) or ((ver.major == 2) and (ver.minor >= 17));
```

Comment:
`Version` supports a comparison method already - use it! I see a few other instances of this too.

also this function should just be merged into `isArchSupportedGlibcVersion`

Reply:
I changed all the version comparisons to use `SemanticVersion.order`.    This function is gone now (more details elsewhere on this review.)